content/en/serverless/installation/dotnet.md

```diff
@@ -91,28 +92,34 @@ More information and additional parameters can be found in [Datadog Serverless C
 {{% /tab %}}
 {{% tab "Serverless Framework" %}}
 
-1. Add the following layers and environment variables to each .NET Lambda function you wish to instrument:
+The [Datadog Serverless Plugin][1] automatically adds the Traceing Layer and environment variables to your functions, and configures your functions to send metrics, traces, and logs to Datadog through the [Datadog Lambda Extension][2].
 
-    ```yml
-    your-function:
-      layers:
-        - arn:aws:lambda:<AWS_REGION>:464622532012:layer:Datadog-Extension:{{< latest-lambda-layer-version layer="extension" >}}
-        - arn:aws:lambda:<AWS_REGION>:464622532012:layer:dd-trace-dotnet:{{< latest-lambda-layer-version layer="dd-trace-dotnet" >}}
-      environment:
-        DD_TRACE_ENABLED: true
-        DD_API_KEY: "<YOUR_DD_API_KEY>"
-        CORECLR_ENABLE_PROFILING: 1
-        CORECLR_PROFILER: "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}"
-        CORECLR_PROFILER_PATH: "/opt/datadog/Datadog.Trace.ClrProfiler.Native.so"
-        DD_DOTNET_TRACER_HOME: "/opt/datadog"
+To install and configure the Datadog Serverless Plugin, follow these steps:
+
+1. Install the Datadog Serverless Plugin:
+    ```sh
+    yarn add --dev serverless-plugin-datadog
+    ```
+2. In your `serverless.yml`, add the following:
+    ```yaml
+    plugins:
+        - serverless-plugin-datadog
     ```
 
-    Replace `<YOUR_DD_API_KEY>` with your Datadog API key, found on the [API Management page][1].
+<div class="alert alert-info">If you are instead deploying your Serverless Framework app <a href="https://www.serverless.com/framework/docs/providers/aws/guide/intro">by natively exporting a JSON object from a JavaScript file</a> (for example, by using a <code>serverless.ts</code> file), follow the <a href="https://docs.datadoghq.com/serverless/installation/dotnet?tab=custom">custom installation instructions</a>.</div>
```

Comment:
Do you think we need to call this specific situation out here? How common is this?